src/components/token/TokenAlerts.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+
+
+const TokenAlerts = (props) => {
+  const [token, setToken] = useState(props.token);
+
+  useEffect(() => {
+    setToken({banned: true});
+  }, [props.token]);
+
+  const verifiedAlert = () => {
+    if (!token.verified) {
+      return null;
+    }
+
+    return (
+      <div className="alert alert-info backup-alert" role="alert">
+        <i className="fa fa-check-circle mr-2" title="Token Verified"></i> This token is verified by Hathor team.
```

Comment:
Maybe put the "Verified" icon next to the name of the token (just like in a Twitter account) and say "This a verified token. <a>Learn more</a>". We can use the same verified icon in the list of tokens inside a transaction.

Reply:
I like it! I'll change and update

Reply:
I agree with that.

I think even verified tokens must continue having the warning rectangle with the token UID message. The only difference should be the verified symbol besides the token symbol with a hover message.

Reply:
Done